examples/nlp_seq/train.py

```diff
@@ -95,8 +80,7 @@
 @functools.partial(jax.jit, static_argnums=(1, 2))
 def create_model(key, input_shape, model_kwargs):
   model_def = models.Transformer.partial(train=False, **model_kwargs)
-  _, initial_params = model_def.init_by_shape(key, [(input_shape, jnp.float32)])
-  model = nn.Model(model_def, initial_params)
+  _, model = model_def.create_by_shape(key, [(input_shape, jnp.float32)])
```

Comment:
create_by_shape is deprecated, please use init_by_shape instead.

Reply:
Done. Thanks!